src/plugins/data/common/search/search_source/search_source.ts

```diff
@@ -412,6 +441,12 @@ export class SearchSource {
           const dataFrameResponse = response as IDataFrameResponse;
           await this.setDataFrame(dataFrameResponse.body as IDataFrame);
           return onResponse(searchRequest, convertResult(response as IDataFrameResponse));
+        } else if ((response as IDataFrameResponse).type === DATA_FRAME_TYPES.POLLING) {
```

Comment:
tbh this might not be as needed since the value gets updated it from the search source's point of view. but lets keep it for now incase there is a mechanism that will end up getting that does something new if polling compared to the default